js/base.tsx

```diff
@@ -64,6 +64,7 @@ document.addEventListener('keydown', e => {
             dialog.close();
         }
         else {
+            dialog.querySelector('form')?.reset();
```

Comment:
That'll just reset the input field right? Not clear the ul too?

Reply:
Yes. Similarly, when you clear the input field yourself, the ul is not cleared either. But maybe whenever the input is empty the ul should be?

Reply:
What does VSCode do? I would expect the dropdown to reflect the state of the input field, like it would with native search as you type like in the referrers dropdown on settings